### PR TITLE
Add a Usage Trends tool

### PR DESCRIPTION
**Pull request by philipwalton**

### CODE REVIEW

File: templates/usage-trends/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <b>Can I use custom dimensions and metrics in a trend report?</b><br>
+    Yes! Just make sure you've selected the property where you've defined the custom dimension or metric.
+  </p>
+  <p>
+    <b>Why are some metrics (e.g. <i>Bounce Rate</i>) missing from the list?</b><br>
+    Usage Trends converts metric totals into percentages of those totals broken down by dimension. Metrics like <i>Bounce Rate</i> that are derived from other metrics (e.g. <i>Bounces</i> / <i>Sessions</i>) are excluded because they wouldn't make sense in a trend report.
+  </p>
+  <p>
+    <b>Why do the results sometimes add up to more than 100%?</b><br>
+    Not all dimensions and metrics can be meaningfully queried together. For example, the <i>User Type</i> dimension is a session-level dimensions (since a user can be a "New Visitor" one sessions and a "Returning Visitor" the next). If you try to create a trend report for the <i>User Type</i> dimension with the <i>Users</i> metric, you'll get unexpected results since some users will be counted twice.
```
Comment (pfrisella) [inline templates/usage-trends/index.html]: s/session-level dimensions/session-level dimension

Comment (pfrisella) [inline templates/usage-trends/index.html]: s/can be a "New Visitor" one sessions/can be a "New Visitor" one session

File: src/javascript/usage-trends/components/view-selector.js [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+      const {account, property, view} = nextState;
+      this.props.onChange({account, property, view});
+    }
+  }
+
+  /** @return {Object} The React component. */
+  render() {
+    if (this.state.hasNoViews) {
+      return (
+        <p>Oops, you don't appear to have any Google Analytics accounts, you
```
Comment (pfrisella) [inline src/javascript/usage-trends/components/view-selector.js]: Side note. Have you tested no GA accounts lately to make sure it's handled correctly?

Comment (philipwalton) [inline src/javascript/usage-trends/components/view-selector.js]: Yeah, I tested it when I staged the site. I assume nothing has changed?

File: templates/usage-trends/index.html (hunk 1 above)
Comment (philipwalton) [inline templates/usage-trends/index.html]: Done.

Comment (philipwalton) [inline templates/usage-trends/index.html]: Done.

**Comment by pfrisella** [comment]
LGTM

**Comment by IronistM** [comment]
Really nice addition @philipwalton ! :+1:

**Pull request merged**